javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointCharacteristics.qll

```diff
@@ -0,0 +1,113 @@
+/**
+ * For internal use only.
+ *
+ * Defines a set of characteristics that a particular endpoint might have. This set of characteristics is used to make
+ * decisions about whether to include the endpoint in the training set and with what label, as well as whether to score
+ * the endpoint at inference time.
+ */
+
+import experimental.adaptivethreatmodeling.EndpointTypes
+import semmle.javascript.security.dataflow.SqlInjectionCustomizations
```

Comment:
Again, what's the advantage of a private import? Is the idea to prevent naming clashes down the line if someone imports this module?

Reply:
(regardless of the answer I think the point is about being consistent here)

Reply:
I private import does not re-export declarations; it makes it so that modules depending on `EndpointCharacteristics.qll` may not depend on (eg.) `SqlInjectionCustomizations`. I'd consider the import of `SqlInjectionCustomizations` as an implementation detail and would therefore default to making it a private import.